chart/values.yaml

```diff
@@ -18,6 +18,9 @@ jwt_token_lifetime_minutes: 1440
 # default time to run behaviors on each page (in seconds)
 default_behavior_time_seconds: 300
 
+# default time to wait for page to fully load before running behaviors (in seconds)
+default_page_load_time_seconds: 120
```

Comment:
The default set here is 120 seconds vs 90 seconds in main.py in the os.environ.get back-up case. Should we be consistent with the default values?

Reply:
Good point - Fixed main.py to also have 120